### PR TITLE
Update UG with enhanced delete

### LINKED ISSUE #107: Update UG for Delete v2
As a user, I need an updated User Guide for the delete command, so that I can make full use of functionalities added to the delete command.

### PR DESCRIPTION
**Pull request by bhnuka**
Update UG with enhanced delete & Separate images into folders

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -178,51 +178,66 @@ Precise expected outputs on failure:
...
-Format: `delete INDEX`
-* Deletes the person at the specified `INDEX`.
+Format: `delete INDEX1 INDEX2 INDEX 3`
+* Deletes the person at `INDEX1`, `INDEX2' and `INDEX3'. (i.e if delete 1 2 3 is input, deletes person at index 1, index 2 and index 3 respectively)
 * The index refers to the index number shown in the displayed person list.
 
 Acceptable parameters for INDEX:
-* The index **must be a positive integer** 1, 2, 3, …​ (less than the size of the contact list in FAPro)
+* The indexes **must be a positive integers** 1, 2, 3, …​ (less than the size of the contact list in FAPro)
+* The indexes must have a space inbetween and no commas (i.e delete 1,2,3 is invalid)
```
Comment (syamfarh) [inline docs/UserGuide.md:332]: I think in between have spaces?
and do you think we should add period for bullet points or nope? I think we should standardize this.

**Review by syamfarh — COMMENTED** [review]
LGTM, just a few nitpicks

File: docs/UserGuide.md (hunk 1 above)
Comment (bhnuka) [reply docs/UserGuide.md:332]: Correct inbetween i have spaces but i was talking more about the individual indexes. I do get the confusion though. Will rectify on my end.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -308,19 +323,19 @@ Format: `find_add KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find_add geylang` returns all users whose addresses contain `geylang`.
 
-![edit format](images/findadd0.png)
+![edit format](images/findadd-UG/findadd0.png)
 
 * Precise expected outputs on success:
 * Successful find message. ‘X persons listed!’’, where X is the number of contacts listed.
```
Comment (glenngnng) [inline docs/UserGuide.md]: Looks like there's a small typo here with two apostrophes for X persons listed! ' '

File: docs/UserGuide.md [hunk 3]
```diff
@@ -278,19 +293,19 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`.
 
-![edit format](images/find0.png)
+![edit format](images/find-UG/find0.png)
 
 * Precise expected outputs on success:
 * Successful find message. ‘X persons listed!’’, where X is the number of contacts listed.
```
Comment (glenngnng) [inline docs/UserGuide.md]: Typo here too! minor nit, nothing much!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -217,34 +217,63 @@ Precise expected outputs on failure:
...
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+#### Format: `delete INDEX1 INDEX2 INDEX 3`
+* Deletes the person at `INDEX1`, `INDEX2' and `INDEX3'. (i.e if delete 1 2 3 is input, deletes person at index 1, index 2 and index 3 respectively)
+* The index refers to the index number shown in the displayed person list
 
-Acceptable parameters for INDEX:
-* The index **must be a positive integer** 1, 2, 3, …​ (less than the size of the contact list in FAPro)
+#### Acceptable parameters for INDEX:
+* The indexes **must be a positive integers** (i.e 1 2 3 etc.) (less than the size of the contact list in FAPro)
```
Comment (glenngnng) [inline docs/UserGuide.md]: Minor typo here, should be must be positive integers

File: docs/UserGuide.md [hunk 5]
```diff
@@ -217,34 +217,63 @@ Precise expected outputs on failure:
...
-![delete format](images/delete-UG/after_delete_success.png)
+```
+Deleted Person(s): X
+```
 
-Precise expected outputs on failure:
-* Error message shown to the user: "The person index provided is invalid".
-* GUI reflects that delete is in red font.
+where X are the details of the deleted persons
+* Size of address book is reduced by however many indexes were entered. (i.e if you entered delete 2 3 4, as you entered 3 indexes, the size of the address book decreases by 3)
```
Comment (glenngnng) [inline docs/UserGuide.md]: i think should remove the full stop here for consistency?

**Review by glenngnng — COMMENTED** [review]
Minor nits, otherwise LGTM

**Comment by glenngnng** [comment]
LGTM!

**Pull request merged**